robot_pose_ekf/CMakeLists.txt

```diff
@@ -66,6 +66,11 @@ install(
     DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
 
+install(
+    DIRECTORY scripts
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
```

Comment:
if so, that will goes to lib/robot_pose_ekf/scripts/wtf.py, do you want to install to lib/robot_pose_ekf/wtf.py ?